pkg/worker/loader.go

```diff
@@ -100,8 +102,8 @@ func (loader *LoaderOSS) Build(ctx context.Context, model *arcadiav1alpha1.Typed
 
 	container := &corev1.Container{
 		Name:            "loader",
-		Image:           "kubeagi/minio-mc:RELEASE.2023-01-28T20-29-38Z",
-		ImagePullPolicy: "IfNotPresent",
+		Image:           loader.worker.Spec.Loader.Image,
```

Comment:
We should use `kubeagi/minio-mc:RELEASE.2023-01-28T20-29-38Z` as the default image. And once user provided a different one,we change it.

Reply:
Keep the default image names